charts/kubechecks/values.yaml

```diff
@@ -28,16 +28,18 @@ deployment:
 
   affinity: {}
 
-  podSecurityContext: {}
-  # fsGroup: 2000
-
-  securityContext: {}
-    # capabilities:
-    #   drop:
-    #   - ALL
-    # readOnlyRootFilesystem: true
-    # runAsNonRoot: true
-    # runAsUser: 1000
+  podSecurityContext:
+    runAsNonRoot: true
+    runAsUser: 32123
+    # fsGroup: 32123
+
+  securityContext:
+    allowPrivilegeEscalation: false
+    privileged: false
+    capabilities:
+      drop:
+        - ALL
+    readOnlyRootFilesystem: true
```

Comment:
I'm not opposed to adding some security defaults, but they would need to be tested first. 
For example, I think setting the FS to RO would break functionality because kubechecks needs to clone repos locally. We could of course have a ephemeral k8s volume where it does this work instead, but I don't think that's in place currently.

Reply:
Fair point. ReadOnlyRootFS is a tricky one. Since it was already "suggested" as a comment in values.yaml, I thought it might be working.